src/main/java/fi/aalto/cs/apluscourses/intellij/utils/CourseFileManager.java

```diff
@@ -75,7 +75,7 @@ public synchronized void createAndLoad(@NotNull URL courseUrl,
    * the course file doesn't exist, {@code true} otherwise.
    *
    * @return {@code true} if the course file was successfully loaded, {@code false} if the course
-   *         file doesn't exist.
+  file doesn't exist.
```

Comment:
This is missing an asterisk at the beginning of the line

Reply:
go try adding it! 😈